### ISSUE TITLE
Toggle for long pressing numbered tiles

### ISSUE THREAD

**Issue by Hedgehog0091** [lucasnlm/antimine-android#280]
tl;dr: read the title
I personally don't like having the option to do so, it allows me to just go through and long press all tiles around the edge and complete the game without doing a whole lot. I think a toggle would be nice, despite what was said in issue #96.
If the 'I don't like it' argument isn't enough, another argument would be someone who doesn't want to open some tiles accidentally does because of a misplaced flag. It just gives the answer without needing to think, and that's no fun.

My idea is either allowing toggling this feature on or off, or setting it between opening all surrounding tiles indiscriminately, flagging or opening tiles if all of them are the same type (current behavior), or disabling this feature entirely.

**Comment by lucasnlm** [comment]
Ok. As a toggle it makes sense.
I'll add it on the next release.

**Comment by lucasnlm** [comment]
A flag button to disable this feature will be added on `10.3.0+`.
For now, I will close this issue. But, feel free to open a new one or reopen this one if needed.
